enterprise/frontend/src/metabase-enterprise/moderation/components/ModerationActions/ModerationActions.jsx

```diff
@@ -13,28 +17,41 @@ ModerationActions.propTypes = {
   onVerify: PropTypes.func,
   moderationReview: PropTypes.object,
   isDataset: PropTypes.bool,
+  renderActions: PropTypes.func,
 };
 
+function defaultRender({ className, isVerified, onVerify, testID }) {
+  return (
+    <Container className={className}>
```

Comment:
any reason we can't simply get rid of this `Container` to avoid the `renderAction`? I'd prefer to avoid the `renderFoo` pattern where possible. perhaps splitting `QuestionModerationSection` into two separate components might make sense